### ISSUE TITLE
don't flux calibrate "undithered dither tiles"

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#1501]
Previously we have been flux calibrating the undithered tile in a set of dither tiles, under the idea that it might be useful.  IIUC the recent dither sequences were off footprint (like tiles 82438 - 82450 on 20211126) and thus didn't have standard stars and thus failed flux calibration creating false pipeline crashes and extra stuff to cleanup.

@schlafly ok if we switch to *not* flux calibrating the "undithered dither tile" by default to avoid this extra distraction?  If you actually have been using the flux calibrated tile then we can continue to try to make them when possible, but if you're not actually using them I'd like to treat them like all the other tiles and only process through sky subtraction.

Q to @schlafly ; implementation coordination with @akremin .

**Comment by schlafly** [comment]
Yes, this isn't critical to the core dither analysis.  The concept was that having tiles with large numbers of bright stars on target could be useful for investigations like Julien's P9 throughput studies.  But we are willing to observe off footprint for the dither analyses and will continue to do so in the future, so it's okay to drop flux calibration here for simplicity.

More broadly, this is probably another case where ideally we'd fall back to a nominal calibration tied to Gaia with a single gray term.
